From 4125a1ccb82fd53d003acdc34e462f238f0c4f0d Mon Sep 17 00:00:00 2001
From: Joe Slater <joe.slater@windriver.com>
Date: Fri, 8 Jul 2022 11:03:22 +0800
Subject: [PATCH] vim: do not adjust script pathnames

When cross-compiling, we do not want to reference the host versions of
things like perl and awk.

Upstream-Status: Pending

Signed-off-by: Joe Slater <joe.slater@windriver.com>
Signed-off-by: Mingli Yu <mingli.yu@windriver.com>
---
 src/Makefile | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/src/Makefile b/src/Makefile
index c9513a632..7a7cbdc43 100644
--- a/src/Makefile
+++ b/src/Makefile
@@ -2552,11 +2552,7 @@ installtools: $(TOOLS) $(DESTDIR)$(exec_prefix) $(DEST_BIN) \
 		 rm -rf $$cvs; \
 	      fi
 	-chmod $(FILEMOD) $(DEST_TOOLS)/*
-# replace the path in some tools
-	perlpath=`./which.sh perl` && sed -e "s+/usr/bin/perl+$$perlpath+" $(TOOLSSOURCE)/efm_perl.pl >$(DEST_TOOLS)/efm_perl.pl
-	awkpath=`./which.sh nawk` && sed -e "s+/usr/bin/nawk+$$awkpath+" $(TOOLSSOURCE)/mve.awk >$(DEST_TOOLS)/mve.awk; if test -z "$$awkpath"; then \
-		awkpath=`./which.sh gawk` && sed -e "s+/usr/bin/nawk+$$awkpath+" $(TOOLSSOURCE)/mve.awk >$(DEST_TOOLS)/mve.awk; if test -z "$$awkpath"; then \
-		awkpath=`./which.sh awk` && sed -e "s+/usr/bin/nawk+$$awkpath+" $(TOOLSSOURCE)/mve.awk >$(DEST_TOOLS)/mve.awk; fi; fi
+# not replace the path in some tools
 	-chmod $(SCRIPTMOD) `grep -l "^#!" $(DEST_TOOLS)/*`
 
 # install the language specific files for tools, if they were unpacked
-- 
2.25.1

